util/configure

```diff
@@ -346,6 +349,13 @@ for my $opt (@ARGV) {
         }
         $path = File::Spec->rel2abs($path);
         push @ngx_opts, "--with-$lib=$path";
+        $ngx_libs{$lib} = "y";
+
+    } elsif ($opt =~ /^--with-(openssl|pcre|zlib)-opt=(.*)/) {
+        my ($lib, $opt) = ($1, $2);
+        $opt.=" -g";
```

Comment:
Missing spaces around binary operator `.=`. Please fix other similar issues in your patch here. I see several such things below as well.